core/src/com/gaalf/presenter/MainMenuPresenter.java

```diff
@@ -17,15 +17,14 @@ public MainMenuPresenter(final GaalfGame game){
     public void startTestLevel(){
         menuMusic.dispose();
 
-//        game.setScreen(new GamePresenter(game, game.levelManager.getLevels().get(0) ));
-        game.playersManager.addPlayer("Jonas", true);
+        game.playersManager.addPlayer("Jonas", true, game.settingsManager.getBallChoice());
         game.devicePlayer = game.playersManager.getPlayers().get(0);
         game.setScreen(new GamePresenter(game, game.levelManager.getRandomLevel()));
     }
 
     public void openLevelSelectMenu() {
-        game.playersManager.addPlayer("Jonas", true);
-        game.playersManager.addPlayer("E", false);
+        game.playersManager.addPlayer("Jonas", true, game.settingsManager.getBallChoice());
+        game.playersManager.addPlayer("E", false, "Square");
```

Comment:
looks confusing. Maybe change the string to "squareBall"?

Reply:
I can change to Squareball, I just thought this looked better

Reply:
To me it looks as if the player is square or something

Reply:
I can change it to square ball if you wanna

Reply:
yes please :)